src/containers/cart/withCart.js

```diff
@@ -430,9 +430,9 @@ export default function withCart(Component) {
                 {(mutationFunction) => (
                   <Component
                     {...this.props}
-                    addItemsToCart={async (items) => {
-                      await this.handleAddItemsToCart(mutationFunction, { items }, !cart);
-                    }}
+                    addItemsToCart={async (items) => (
+                      this.handleAddItemsToCart(mutationFunction, { items }, !cart)
```

Comment:
If we're no longer awaiting the result of `handleAddItemsToCart` then `async` should also be removed

Reply:
I removed the `async` not sure why it's not reflected in the comment above.